assets/src/dashboard/app/views/shared/storyListView.js

```diff
@@ -83,7 +83,7 @@ const PreviewContainer = styled.div`
   height: ${({ theme }) => theme.previewWidth.thumbnail / FULLBLEED_RATIO}px;
   vertical-align: middle;
   border-radius: ${({ theme }) => theme.storyPreview.borderRadius}px;
-  border: ${({ theme }) => theme.storyPreview.border};
+  border: ${({ theme }) => theme.borders.gray75};
```

Comment:
noticed one border theme setting that wasn't part of the `borders` key, updated the few places it's used to make those more streamlined.